tgrsite/templates/tgrsite/addnavbar.html

```diff
@@ -157,6 +157,7 @@
 
 				{% endcomment %}
 				{% if request.user.is_authenticated %}
+					<li><a href="{% url 'all_notifications' %}"><i class="fa fa-bell" aria-hidden="true"></i> {{request.user.member.notification_count}}</a></li>
```

Comment:
Since this is a link to a page, I think we shouldn't specify `aria-hidden="true"` and instead add `aria-label="Notifications"` or similar

Ref: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Reply:
Good catch, :dango: